src/many-identity-webauthn/src/identity/u2fhid.rs

```diff
@@ -0,0 +1,421 @@
+//! This is a copy of the webauthn-authenticator-rs u2fhid implementation,
+//! with a difference in the API; the perform_auth function returns a
+//! PublicKeyCredentials which has no rawId in it, as the Ledger Nano
+//! doesn't have proper CTAP 2.1 support (notably the attestation object
+//! is missing).
+//! We only use Attestation to get the rawId, and we (MANY) don't use the
+//! rawId at all, so skipping it does nothing and fixes a support bug.
```

Comment:
Interesting, can you link to some issue/documentation about this?

Also, can you highlight the changes you made in this file?